react/features/billing-counter/middleware.js

```diff
@@ -11,6 +11,9 @@ import { setBillingId } from './functions';
  * @param {Store} store - The redux store.
  * @returns {Function}
  */
+
+let wasCounted = false;
```

Comment:
Why not put this in redux?

Reply:
This part of the functionality does not have its own reducer. Adding it would imply creating a new reducer which would only be used for storing this flag. (I don't know if it's worth it. We could add it if we see there is need for it in other parts of the application.)

Secondly, having this in redux, would trigger an unnecessary execution of `mapStateToProps` across the app.

WDYT? 🤔

Reply:
True, I totally agree with your arguments, but they are all true for any action or anything that we store in redux. So I kind of classify them as general downsides of redux and arguments against using redux in Jitsi Meet which is whole another story.

IMHO since we are currently using redux and we are even trying to migrate the legacy code to use React/Redux, we should use it even for those simple cases. Also I don't think creating a reducer will be too much of a work and I'm not too worried about one unnecessary execution of mapStateToProps across the app (at the end of the day this is the price we pay to use redux).

Another idea, that is not directly related to your changes but it is related to the feature, is to store the billing id in redux too and take advantage of the persisting redux store scheme instead of managing localStorage by yourself.

Reply:
Fair enough. I have added the flag to the store.